src/Tizen.Wearable.CircularUI.Chart.Forms.Renderer/BarChartViewRenderer.cs

```diff
@@ -0,0 +1,752 @@
+                    else if( barDataSet != null && barDataSet.BarBackgroundColor != Color.Default)
+                    {
+                        barBGColor = SkiaSharp.Views.Forms.Extensions.ToSKColor(barDataSet.BarBackgroundColor);
+                    }
+                    else
+                    {
+                        var barColor = entry.Color;
+                        barColor = barColor == Color.Default ? BarDefaultColor : barColor;
+                        barBGColor = SkiaSharp.Views.Forms.Extensions.ToSKColor(barColor).WithAlpha(BarBackgroundAlpha);
+                    }
+
+                    using (var paint = new SKPaint
+                    {
+                        Style = SKPaintStyle.Fill,
+                        Color = barBGColor,
+                    })
+                    {
+                        var x = IsVertical ? point.X - (barSize.Width / 2) : _majorAxisSize.Width ;
+                        var y = IsVertical ? 0 : point.Y - (barSize.Width / 2);
+                        var w = IsVertical ? barSize.Width : barSize.Height;
+                        var h = IsVertical ? barSize.Height : barSize.Width;
+                        rect = SKRect.Create(x, y, w, h);
+                        if (topRadius > 0)
+                        {
+                            canvas.DrawRoundRect(rect, topRadius, topRadius, paint);
+                        }
+                        else
+                        {
+                            canvas.DrawRect(rect, paint);
+                        }
+                    }
+                    index++;
+                }
+            }
+        }
+
+        protected void DrawBars(SKCanvas canvas, SKPoint[] points, SKSize barSize)
+        {
+            var halfWidth = barSize.Width / 2;
+            var topRadius = (float)Element.BarTopRadius > halfWidth ? halfWidth : (float)Element.BarTopRadius;
+            var bottomRadius = (float)Element.BarBottomRadius > halfWidth ? halfWidth : (float)Element.BarBottomRadius;
+            bool isValueOverRadius = false;
+
+            if (points.Length > 0)
+            {
+                if (Element.BarChartType == BarChartType.Vertical)
+                {
+                    var dataSet = Element.Data[0];
+                    for (int i = 0; i < _categoryCount; i++)
+                    {
+                        if (_barChartDataTable[i] == null)
+                            continue;
+
+                        var point = points[i];
+                        var entry = _barChartDataTable[i];
+                        Color barColor = entry.Color != Color.Default ? entry.Color : dataSet.Color != Color.Default ? dataSet.Color : BarDefaultColor;
+                        using (var paint = new SKPaint
+                        {
+                            Style = SKPaintStyle.Fill,
+                            Color = SkiaSharp.Views.Forms.Extensions.ToSKColor(barColor),
+                        })
+                        {
```

Comment:
Why not use DrawLine with `StrokeWidth = barSize.Width`?

Reply:
Is it possible to draw rounded line using DrawLine. I only used DrawRect and DrawRoundRect for drawing radius bar.

Reply:
Yes, you can draw rounded line with SKStrokeCap.Round